dev/tools/localization/gen_l10n.dart

```diff
@@ -836,6 +596,39 @@ class LocalizationsGenerator {
     supportedLocales.addAll(localeInfoList);
   }
 
+  static bool _isValidGetterAndMethodName(String name) {
+    // Dart getter and method name cannot contain non-alphanumeric symbols
+    if (name.contains(RegExp(r'[^a-zA-Z_\d]')))
+      return false;
+    // Dart class name must start with lower case character
+    if (name[0].contains(RegExp(r'[A-Z]')))
```

Comment:
Or underscore?

Reply:
In this case (method) name must work as a public identifier so there shouldn't be a leading underscore. Added a check for that.